packages/service/ni_measurement_plugin_sdk_service/_internal/grpc_servicer.py

```diff
@@ -363,5 +386,11 @@ def Measure(  # noqa: N802 - function name should be lowercase
 
     def _serialize_response(self, outputs: Any) -> v2_measurement_service_pb2.MeasureResponse:
         return v2_measurement_service_pb2.MeasureResponse(
-            outputs=_serialize_outputs(self._output_metadata, outputs)
+            outputs=_serialize_outputs(
+                self._output_metadata, outputs, self._get_service_name() + ".Outputs"
+            )
         )
+
+    def _get_service_name(self) -> str:
```

Comment:
Don't duplicate this method. Make a free function at top level so both classes can use it.